config_validation_engine/validators/tests/test_validate.py

```diff
@@ -1,11 +1,29 @@
 from config_validation_engine.validators import validators as val
+import unittest
+import yamale
 
-def test_email():
-    v = val.Email()
-    assert not v.is_valid('random@')
 
-def test_latitude():
-    v = val.Latitude()
-    assert v.is_valid(23)
-    assert not v.is_valid(23.1)
-    
\ No newline at end of file
+class MyTest(unittest.TestCase):
+
+    def test_email(self):
+        v = val.Email(self)
+        assert not v.is_valid('random@')
+
+    def test_latitude(self):
+        v = val.Latitude()
+        assert v.is_valid(23)
+        assert v.is_valid(23.1)
+
+    def test_URL(self):
+        v = val.URL()
+        assert not v._is_valid('test.com')
+        assert v._is_valid('http://test.com')
+
+    def test_site_object(self):
+        schema = yamale.make_schema('./schema/site_object_schema.yaml', validators=val.all_config_validators())
```

Comment:
We run pytest from the root of the git repository. Can you change the path of the schema file to:

```
./config_validation_engine/validators/schema/site_object_schema.yaml
```